hm-module/monitors.nix

```
@@ -56,29 +56,36 @@ in {
             '';
           };
           position = lib.mkOption {
-            type = types.either (point2DType types.ints.unsigned)
-              (types.enum [ "auto" ]);
+            type = types.either (point2DType types.ints.unsigned) (types.enum [
+              "auto"
+              "auto-up"
+              "auto-down"
+              "auto-left"
+              "auto-right"
+              "auto-center-up"
+              "auto-center-down"
+              "auto-center-left"
+              "auto-center-right"
+            ]);
             default = "auto";
             description = ''
-              The position of the monitor as `{ x, y }` attributes.
-
-              This is not the same as {option}`position` which is a
-              string or an enum string.
+              The position of the monitor as `{ x, y }` attributes,
+              offset relative to the top-left corner of virtual screen space.
             '';
           };
           resolution = lib.mkOption {
             type = types.either (point2DType types.ints.positive)
-              (types.enum [ "preferred" "highrr" "highres" ]);
+              (types.enum [ "preferred" "highrr" "highres" "maxwidth" ]);
             default = "preferred";
             description = ''
               The physical size of the display as `{ x, y }` attributes.
 
-              This is not the same as {option}`option` which is a
-              string or an enum string.
+              Note, if you want define your `position` attributes relative to
+              each other, use the value of {option}`scale` recursively.
```

```suggestion
              ::: {.note}
              If you want define your `position` attributes relative to
              each other, use the value of {option}`scale` recursively.
              :::
```

It's debatable whether this is necessary, but it'll do a good job at highlighting the note if we ever decide to add rendered documentation. Hint hint [ndg](https://github.com/feel-co/ndg) wink wink.

---

My order of attack is:
- functional improvements/parity
- improve organization where apparent
- generate static documentation
- improve documentation

I'll make a note of the syntax, but you'll have more chances to review documentation in Phase 3.

---

It's nixpkgs-flavored commonmark, it is suported by both nixos-render-docs and ndg.